src/server.h

```diff
@@ -41,6 +41,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <map>
 #include <vector>
 
+extern bool no_asciiart;
```

Comment:
This is better solved without a global variable.
`g_settings->setDefault()` might be helpful

Reply:
That global variable is to pass the state of the command line modifier --no-asciiart and I didn't mean that modifier to change the settings, only that it deactivate the logs once.
To deactivate it persistently, there is the other setting from the menu or settings file.
Anyway, the use of that global variable was not a correct way to do it, simple but not correct.